app/steps/ui/will/codicils/index.js

```diff
@@ -13,20 +14,34 @@ class WillCodicils extends ValidationStep {
         return this.next(ctx).constructor.getUrl('codicils');
     }
 
-    nextStepOptions() {
-        const nextStepOptions = {
+    handlePost(ctx, errors, formdata, session, hostname, featureToggles) {
+        ctx.isToggleEnabled = FeatureToggle.isEnabled(featureToggles, 'screening_questions');
+
+        return [ctx, errors];
+    }
+
+    nextStepOptions(ctx) {
+        if (ctx.isToggleEnabled) {
+            return {
+                options: [
+                    {key: 'codicils', value: json.optionNo, choice: 'noCodicilsToggleOn'}
+                ]
+            };
+        }
```

Comment:
Can we have a unit test for this new method

Reply:
This method should also have a unit test for the case where the toggle is enabled as this will output a different return value